newapi/ooniapi/measurements.py

```diff
@@ -1166,6 +1166,16 @@ def get_aggregated() -> Response:
         description: >-
           The end date of when measurement were run (ex.
           "2016-10-20T10:30:00")
+      - name: time_gran
```

Comment:
I think the name of this argument could be improved. `time_gran` is not particularly understandable and the doc string "Time granularity" doesn't really do much to help explain how it will affect the output data.

Maybe it could be called "tick_every" (which is commonly used in charting applications to indicate the ticks on the axis).

Alternatively it could perhaps simply be called "group_by"?

Reply:
It's described in the "description:" field. It contains "time_" to clarify that it's related specifically to time. I added a better description.

Reply:
We discussed out of band that maybe a name that preserves both being short and understandable is `time_grain`, so let's go for that.

Reply:
Moved to time_grain